Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/ReflectedPropertyEditor.cpp

```diff
@@ -606,7 +626,7 @@ namespace AzToolsFramework
     // creates and populates the GUI to edit the property if not already created
     void ReflectedPropertyEditor::Impl::CreateEditorWidget(PropertyRowWidget* pWidget)
     {
-        if (!pWidget->HasChildWidgetAlready())
+        if ((!pWidget->HasChildWidgetAlready()) && (!pWidget->GetToggle()))
```

Comment:
nit: additional parens are not necessary around the expressions, seems to make the code a little too noisy

Reply:
Good catch!